src/main/java/org/spongepowered/mod/mixin/entity/player/MixinEntityPlayer.java

```diff
@@ -71,4 +75,8 @@ public MixinEntityPlayer(World worldIn) {
         return this.openContainer == this.inventoryContainer;
     }
 
+    public String human$getName() {
+        return this.getName();
```

Comment:
Actually just for the record, the way @Aaron1011 has done this is exactly correct. By placing the `@Overwrite` with the original code *below* the interface accessor, it will *not* cause the infinite loop. However @simon816 is absolutely correct, this can now be solved with [`@Intrinsic`](https://github.com/SpongePowered/Mixin/blob/master/src/main/java/org/spongepowered/asm/mixin/Intrinsic.java) because the purpose of `Intrinsic` is precisely to avoid this kind of unnecessary duplication.

So basically, this code is quite correct - per my examples - but using `Intrinsic` will make it better.